src/main/kotlin/org/rekotlin/BlockSubscriber.kt

```diff
@@ -0,0 +1,16 @@
+package org.rekotlin
+
+/**
+ * BlockSubscriber allows subscription to a single state. This methodology is useful when subscribing multiple states
```

Comment:
Please rephrase the comments, 
it should be something like below...
BlockSubscriber allows subscribing to multiple states in a single class.

Reply:
Updated